src/commonmark/editor.ts

```diff
@@ -8,6 +8,7 @@ import {
     commonmarkImageUpload,
     defaultImageUploadHandler,
 } from "../shared/prosemirror-plugins/image-upload";
+import { placeholderPlugin } from "../rich-text/plugins/placeholder";
```

Comment:
Since this is used in both editors, we should move this file to `/shared/prosemirror-plugins`